### DISCUSSION TITLE
How to connect with MariaDB

### DISCUSSION THREAD

**Discussion by philipphagen87** [linq2db/linq2db#4123] [Q&A] [answered]
Hi all,

it's not possible for me to connect to maria DB for me :-(
This is my setting
![image](https://github.com/linq2db/linq2db/assets/43008517/8d23d607-57f6-4801-b916-80466dc53d77)

this is the issue that occure
![image](https://github.com/linq2db/linq2db/assets/43008517/d3495f17-d800-4723-8969-d323380e2183)

that happens in this line:
![image](https://github.com/linq2db/linq2db/assets/43008517/0d6c5316-1949-4279-8121-40b2b16aedf1)

I tried the connection via "normal" mysql connector, it works so from the connectionstring side should everything be good.

Thanks for helping :-)

**Comment by MaceWindu** [comment, marked as answer]
use `ProviderName.MySqlConnector` instead of `ProviderName.MariaDB` (#4124)
